src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -631,7 +631,7 @@ Parameter Engine::GetMetric(const std::string& name, const std::map<std::string,
         const std::tuple<unsigned int, unsigned int> range = std::make_tuple(1, parallel_get_max_threads());
         return decltype(ov::range_for_streams)::value_type(range);
     } else if (name == ov::caching_properties) {
-        std::vector<ov::PropertyName> cachingProperties;
+        std::vector<ov::PropertyName> cachingProperties = { ov::device::full_name };
```

Comment:
CPU device itself should always not impact cache hash computation due to there is no multiple CPU devices, right?
So for CPU, I think we don't need add this additional ov::device::full_name to ov::caching_properties.

Reply:
discussed offline:

if we remove ov::device::full_name, it means CPU compiled blob will not contain any information that it's CPU specific
we need at least one property to identify blob to CPU